lib/gds-components/autocomplete/index.d.ts

```diff
@@ -0,0 +1,210 @@
+
```

Comment:
I have taken this type definition from [here](https://github.com/alphagov/accessible-autocomplete/issues/535)